clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/agent/KubernetesV2CachingAgentDispatcher.java

```diff
@@ -80,20 +80,18 @@ public KubernetesV2CachingAgentDispatcher(ObjectMapper objectMapper, Registry re
                     .filter(Objects::nonNull)
                     .forEach(c -> result.add((KubernetesCachingAgent) c)));
 
-    if (v2Credentials.isMetricsEnabled()) {
-      IntStream.range(0, credentials.getCacheThreads())
-          .boxed()
-          .forEach(
-              i ->
-                  result.add(
-                      new KubernetesMetricCachingAgent(
-                          credentials,
-                          objectMapper,
-                          registry,
-                          i,
-                          credentials.getCacheThreads(),
-                          agentInterval)));
-    }
+    IntStream.range(0, credentials.getCacheThreads())
+        .boxed()
```

Comment:
while you're in here, remove the call to boxed?

Reply:
:+1: 
:no_entry_sign: :package: